core/src/main/java/fr/sncf/osrd/conflicts/Conflicts.kt

```diff
@@ -204,6 +241,119 @@ class IncrementalConflictDetectorImpl(trainRequirements: List<TrainRequirements>
         }
         return res
     }
+
+    override fun minDelayWithoutConflicts(
+        spacingRequirements: List<SpacingRequirement>,
+        routingRequirements: List<RoutingRequirement>,
+        globalMinDelay: Double
+    ): Double {
+        var minDelay = Double.POSITIVE_INFINITY
+        var hasConflict = false
+        for (spacingRequirement in spacingRequirements) {
+            if (spacingZoneRequirements[spacingRequirement.zone!!] != null) {
+                val conflictingRequirements =
+                    spacingZoneRequirements[spacingRequirement.zone!!]!!.filter {
+                        !(spacingRequirement.beginTime >= it.endTime ||
+                            spacingRequirement.endTime <= it.beginTime)
+                    }
+                if (conflictingRequirements.isNotEmpty()) {
+                    hasConflict = true
+                    val latestEndTime = conflictingRequirements.maxOf { it.endTime }
+                    minDelay = min(minDelay, latestEndTime - spacingRequirement.beginTime)
```

Comment:
This is supposed to be a `max` (the minimum delay is the maximum delta)

The initial value can be 0 instead of +inf, and we don't need `hasConflict` anymore